.github/workflows/linting-and-tests.yml

```diff
@@ -262,15 +262,21 @@ jobs:
           pytest -x
 
   end-to-end-tests:
-    # TODO: reenable this job once https://github.com/grafana/oncall/issues/1692 is fixed
-    if: ${{ false }}
     runs-on: ubuntu-latest
     name: "End to end tests - Grafana: ${{ matrix.grafana-image-tag }}"
     strategy:
       matrix:
         grafana-image-tag:
-          - 8.5.22
-          - 9.2.6
+          # OnCall doesn't work on the following versions of Grafana
+          # - 8.5.22
+          # - 9.0.0
+          # - 9.1.0
```

Comment:
@Ukochka will your PR (#1800) fix these issues? If so, you should consider turning on the tests for these versions in your PR